src/elements/content-sidebar/activity-feed/annotation-thread/AnnotationThreadContent.js

```diff
@@ -54,6 +55,16 @@ const AnnotationThreadContent = ({
         errorCallback: onError,
     });
 
+    const onReplyEditHandler = (
+        id: string,
+        text: string,
+        status?: FeedItemStatus,
+        hasMention?: boolean,
+        replyPermissions: BoxCommentPermission,
+    ) => {
+        handleEditReply(id, text, status, replyPermissions);
```

Comment:
Minor, but it looks like the naming got a bit twisted up with `useAnnotationAPI`. We'd typically expect callback props to use "on" while internal handlers use "handle", so this is inverted.

Reply:
I agree, the naming of those handleXYZ functions in `useAnnotationAPI` could be changed to onXYZ, but I think it should be done in a separate PR. Here I only wanted to fix the issue we had with passing `handleEditReply` (without the `hasMention` as 4th argument) directly to  `onReplyEdit` prop of `ActivityThread`.